release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -192,6 +192,22 @@ installation does not fail {product-title} cluster installation.
 You can xref:../openshift_images/samples-operator-alt-registry.adoc#installation-restricted-network-samples_samples-operator-alt-registry[configure alternate or mirrored registries]
 to bypass this issue if it arises during your cluster installation.
 
+[id="ocp-4-4-developer-experience"]
+=== Developer experience
+
+[id="ocp-4-4-odo"]
+==== odo enhancements
+
+`odo` has a few enhancements and improvements that focus on the user experience:
+* An `odo debug info` command is added.
+* The `odo url` command now has a `--secure` flag to specify HTTPS URLs.
+* The `odo create`, `odo url`, and `odo config` commands now have a `--now` flag to apply changes on the cluster immediately.
+* The `odo debug port-forward` command now selects a port automatically if the default port is occupied.
+* Enhancement of output of  the `odo storage` and `odo push` commands.
```

Comment:
Suggest making these last 3 bullets full sentences to be consistent with the rest of the bullets.